### PR TITLE
set viewport

### PR DESCRIPTION
**Pull request by abjjabjj**
so mobile viewing is better

### CODE REVIEW

File: webapp/main/templates/index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
 
 <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
[the remark this answers is not in the document]
Comment (abjjabjj) [reply webapp/main/templates/index.html:5]: Do you mean setting `maximum-scale` to achieve a max size cutoff?

[the remark this answers is not in the document]
Comment (MichaelDeng03) [reply webapp/main/templates/index.html]: Link CSS is currently using absolute sizing, which won't be affected by viewport. Could you change link box sizing to some relative measure like rem/vw? You could also set a maximum size or an @media rule to change sizing based on device size - that way, we can keep the links relatively small on desktop, but comparatively larger on mobile.

[the remark this answers is not in the document]
Comment (abjjabjj) [reply webapp/main/templates/index.html]: kk, gimme a sec to make the changes [](edited-with-Axolo)

**Comment by abjjabjj** [comment]
@MichaelDeng03 lmk if you find the styling up to par

**Comment by MichaelDeng03** [comment]
looks sexy 👍

**Pull request merged**